src/core/cff_parser.js

```diff
@@ -1383,7 +1383,11 @@ class CFFCompiler {
       data: [],
       length: 0,
       add(data) {
-        this.data = this.data.concat(data);
+        if (data.length <= 65536) {
```

Comment:
Maybe add a short comment mentioning why the limit is used here, to prevent something from trying to just remove that?